js/bybit.js

```diff
@@ -600,6 +629,17 @@ module.exports = class bybit extends Exchange {
                     'OMNI': 'OMNI',
                     'SPL': 'SOL',
                 },
+                'networksById': {
```

Comment:
@ttodua isn't this usually called `reverseNetworks` ?

Reply:
> `reverseNetworks`

No, `reverseNetworks` is not same, moreover it's almost nowhere used - only in binance & tokocrypto (which is based on  purely binance-class almost) and they contain different data. (additionally, mistakenly someone applied 'reverseNetworks' also in wavesexchange).  

in all other exchanges, if you search for word `networksById` you will find out that is exactly what I refer too, the difference is that in all other exchanges `networksById` variable is hardcoded, while I mentioned on discord that 'hardcoded' approach is not ideal and we should move them in `networksById` field under options, the next sibling where to `networks` key. 

additional note, `networksById` is not just reversed key-values of `networks` object, because `networks` values have many-to-one relation, for example it might be such (let's say 'Ethereum' and 'TronNetwork' are exchange-specific slugs for chains used in withdrawal/deposit network actions):
```
'networks': {
    'ERC20': 'Ethereum',
    'ETH': 'Ethereum',
    'TRX': 'TronNetwork',
    'TRON': 'TronNetwork',
    'TRC20': 'TronNetwork',
    ...
}
```

however, `networksById` variable (as you see across exchange implementations) is reversed, but with one-to-one relation. (and I see that CCXT has one chosen name for each popular network, for example, `TRC20` seems the chosen core networkCode for tron, `ERC20` for ethereum...)